### DISCUSSION TITLE
A question to get intuition

### DISCUSSION THREAD

**Discussion by MaryGoAround** [GreenleafLab/ArchR#759] [Questions / Documentation]
Hello

Indeed, this is not an issue with your software, rather my question please;

If I am not wrong, let's see, cell-ranger pipeline already runs macs2 and generate fragment file

Then, what is the point to peak calling by ArchR once again?

Sorry if my question seems stupid

I also after peak calling steps get this plot instead of gene name with chromosomes 

![Rplot07](https://user-images.githubusercontent.com/60115705/118427037-3f7e0200-b6c4-11eb-9529-212c9e0e9b8c.png)

**Comment by rcorces** [comment]
Please note that Issues are reserved for bug reports. questions about usage should be posted in Discussions as stated in the Issue template. I have converted this for you.

**Comment by rcorces** [comment]
cellranger-atac does not use MACS2 to create the fragments file. This is created directly from the bam file.

Your second question is not entirely clear but I would ask why you are expecting to get genes when you are plotting a heatmap of the peak matrix.
